src/common.js

```diff
@@ -157,6 +158,8 @@ function ShellString(stdout, stderr, code) {
 
 exports.ShellString = ShellString;
 
+// DEPRECATED: Use os.homedir instead
```

Comment:
Don't deprecate internal functions, just remove them. We only explicitly export the functions listed here:

https://github.com/shelljs/shelljs/blob/master/plugin.js#L7